pyaerocom/colocation/colocation_setup.py

```diff
@@ -463,6 +463,36 @@ def validate_no_forbidden_keys(self):
         for key in self.FORBIDDEN_KEYS:
             if key in self.model_fields:
                 raise ValidationError
+        return self
+
+    @model_validator(mode="after")
+    # @classmethod
+    def validate_start_stop_xand(self):
+        if not (self.start and self.stop):
+            if self.start or self.stop:
+                raise ValueError("Both start and stop need to be provided or both not provided.")
+        return self
+
+    # @model_validator(mode="after")
```

Comment:
Can we not just leave this in?

Reply:
We can. But then we have to remove the `return self` above. I'm not sure if this might lead to other strange behavior